dls-controls/malcolmjs
# Feature/prevent link overwrites #210

## Description

Fixed the performance issues by processing attribute updates in bulk.
Various style changes.
Bought some consistency to test file structures.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Start the site and see the CPU usage.

## Agile board tracking

connect to #268
